runconfig/parse_windows.go

```diff
@@ -0,0 +1,20 @@
+package runconfig
+
+import (
+	"fmt"
+	"strings"
+)
+
+func parseNetMode(netMode string) (NetworkMode, error) {
+	parts := strings.Split(netMode, ":")
```

Comment:
Purely to keep it in sync with Linux (for now). You're right, none of this does anything quite yet - I can remove that once we have real code in there. I just wanted to get us off overloading 'bridge' in the first instance.